lib/main.dart

```diff
@@ -44,6 +45,11 @@ class MyApp extends StatelessWidget {
       initialRoute: '/username_add',
       title: "StarBook",
       theme: c.CupertinoThemeData(),
+      localizationsDelegates: [
```

Comment:
why do we need delegates?

Reply:
Sir, if delegates are not used then the app crashes whenever we long press on SelectableText widget to copy the text. 

SelectableText widget requires either MaterialApp as root or MaterialLocalizations delegate 
and it does not require the other two delegates WidgetLocalization and CupertinoDelegates

It shows the following error
```
The following assertion was thrown building _OverlayEntryWidget-[LabeledGlobalKey<_OverlayEntryWidgetState>#e41b0](dirty, dependencies: [_LocalizationsScope-[GlobalKey#2d773]], state: _OverlayEntryWidgetState#0bfc9):
No MaterialLocalizations found.

_OverlayEntryWidget widgets require MaterialLocalizations to be provided by a Localizations widget ancestor.
The material library uses Localizations to generate messages, labels, and abbreviations.

To introduce a MaterialLocalizations, either use a MaterialApp at the root of your application to include them automatically, or add a Localization widget with a MaterialLocalizations delegate.

The specific widget that could not find a MaterialLocalizations ancestor was: _OverlayEntryWidget-[LabeledGlobalKey<_OverlayEntryWidgetState>#e41b0]
  dirty
  dependencies: [_LocalizationsScope-[GlobalKey#2d773]]
  state: _OverlayEntryWidgetState#0bfc9
The ancestors of this widget were: 
  : CupertinoApp
    state: _CupertinoAppState#fdf61
  : MyApp 
```


![WhatsApp Image 2021-01-19 at 21 01 48](https://user-images.githubusercontent.com/55757584/105059947-daabf580-5a9d-11eb-84fd-bffa8a9a3515.jpeg)